R/gh_response.R

```diff
@@ -21,7 +21,7 @@ gh_process_response <- function(response) {
       headers = heads,
       message = paste0("GitHub API error (", status_code(response), "): ",
                        heads$`status`, "\n  ", res$message, "\n")
-    ), class = c("condition", "error"))
+    ), class = c("github_error", paste0("http_error_", status_code(response)), "error", "condition"))
```

Comment:
This was already added by Jim's PR, no? Can you please rebase, if it is easy? If it is not easy, then don't worry about it, it is not worth getting into git trouble for this.....